deploy-as-code/helm/charts/digit-works/frontend/works-ui/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
+name: works-ui
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.16.0
+
+dependencies:
+- name: common
+  version: 0.0.5
+  repository: file://../../../common
```

Comment:
I think the common directory is present one level above this. 
We will need an extra `../`. 
Please verify and make the change.

Reply:
Yeah i have added extra